frontend/dispatcher.lua

```diff
@@ -337,6 +341,12 @@ function Dispatcher:init()
     for i=1,#CreOptions do
         parseoptions(CreOptions, i)
     end
+
+    option_text_table = {}
+    for i,val in pairs(settingsList) do
+        option_text_table[tostring(val.event)] = val.toggle
```

Comment:
This will also need to be done in `Dispatcher:registerAction`

Reply:
If we register an Action, do we have a `toggle` field there?
I included
```
        for i, j in pairs(value) do
            print("xxxxxx "..i .. " type:" .. type(j))
        end
```
and found no toggles (tables) so far.


But I will change `registerAction` to (maybe the will come an plugin, which uses that) 
```
function Dispatcher:registerAction(name, value)
    if settingsList[name] == nil then
        settingsList[name] = value
        if value.event and value.toggle then
            option_text_table[value.event] = value.toggle
        end
        table.insert(dispatcher_menu_order, name)
    end
    return true
end
```